tests/libvmi/vmi_test.go

```diff
@@ -0,0 +1,39 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright the KubeVirt Authors.
+ *
+ */
+
+package libvmi
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("libvmi annotations", func() {
```

Comment:
There should be no unit tests for this library.
1. One can use unit tests for development, but merging unit tests for e2e test libs is not something I want to maintain.
2. There is no real logic to this vmi builder, and if there is, it should not be placed here.
3. We could take this out to the `pkg` folder or to a separated repo and then test it freely.